src/sentry/integrations/vsts/issues.py

```diff
@@ -306,15 +306,16 @@ def sync_status_outbound(
                 },
             )
 
-    def should_unresolve(self, data: Mapping[str, str]) -> bool:
-        done_states = self.get_done_states(data["project"])
-        return not data["new_state"] in done_states or data["old_state"] is None
-
-    def should_resolve(self, data: Mapping[str, str]) -> bool:
-        done_states = self.get_done_states(data["project"])
-        return not data["old_state"] in done_states and data["new_state"] in done_states
+    def get_resolve_sync_action(self, data: Mapping[str, Any]) -> ResolveSyncAction:
+        done_states = self._get_done_statuses(data["project"])
+        return ResolveSyncAction.from_resolve_unresolve(
+            should_resolve=(
+                not data["old_state"] in done_states and data["new_state"] in done_states
+            ),
+            should_unresolve=(not data["new_state"] in done_states or data["old_state"] is None),
+        )
 
-    def get_done_states(self, project: str) -> Sequence[str]:
+    def _get_done_statuses(self, project: str) -> Set[str]:
```

Comment:
should this be `_get_done_states`?

Reply:
I wanted to match `_get_done_statuses()` from `src/sentry/integrations/jira/`.